config.yaml

```diff
@@ -4,13 +4,12 @@
 # If your charm does not require configuration options, delete this file entirely.
 #
 # See https://juju.is/docs/config for guidance.
-
 options:
-  # An example config option to customise the log level of the workload
-  log-level:
-    description: |
-      Configures the log level of gunicorn. 
-
-      Acceptable values are: "info", "debug", "warning", "error" and "critical"
-    default: "info"
+  tls-secret-name:
```

Comment:
I'm not sure we need this, do we? Since we're providing the `tls-certificates` relation the charm can define this internally. I believe this option is only needed if we plan to support a different approach to managing certificates, and need to manually specify the name of the secret that will be used.

Reply:
Make sense, I don't see any real use case for `tls-secret-name` anyways so I removed that config option. Will add it again if it's asked for